test/lint.ts

```diff
@@ -0,0 +1,6 @@
+export * from "../src/lint";
+import * as Test from "./helper";
+if (false) {
+    console.log(Test);
+}
```

Comment:
No.... this is to satisfy tslint itself. `Test` is flagged as an unused variable (since it's only used in the `export {Test}` below, yes, there's already an open issue on this), by having this bit of unreachable code, the erroneous error is squelched. Though I could probably have squelched it with a comment instead.